### DISCUSSION TITLE
Refresh RIR Database

### DISCUSSION THREAD

**Discussion by Danielsc87** [HomeITAdmin/nextcloud_geoblocker#111] [General]
Hello,

I have installed and using RIR Database, everything is working well, tested with VPN and it blocked every country I moved the VPN to and got the correct country code. Nice work,

Just wondering how often the Database would need to be Refreshed, or how often do IP addresses change country's.

**Comment by HomeITAdmin** [comment]
Hi, 
the RIRs are updating them every day. So to be sure you have to update every day. But the question is, how likely it is, to run into a just changed IP in the wild and in addition the change makes a difference for your blocking. In addition we also had an issue a few month ago, that the list from the RIR was actually not correct. So it is not perfect any how.
I personally feel good with updating once a month.

**Reply by Danielsc87** [reply]
Hi Thank you for the reply. Is there a way to do an schedule an update.

**Reply by HomeITAdmin** [reply]
You can run the update via occ on the command line. You can trigger this e.g. via cron job regularly.

**Reply by Danielsc87** [reply]
Do you have the occ command I can't find it.

**Reply by HomeITAdmin** [reply]
As a hint, when you call occ without a command you get a list of all available commands.
To update the db of the currently selected service you can call `occ geoblocker:localization-service:update-db`

**Reply by Danielsc87** [reply]
Nice thank you, I didn't know you could do that with the occ commands. I'll keep that in mind thank you.
